subsys/bluetooth/host/iso.c

```diff
@@ -592,7 +592,7 @@ void bt_iso_recv(struct bt_conn *iso, struct net_buf *buf, uint8_t flags)
 	BT_ISO_DATA_DBG("handle %u len %u flags 0x%02x pb 0x%02x ts 0x%02x",
 			iso->handle, buf->len, flags, pb, ts);
 
-	/* When the PB_Flag does not equal 0b00, the fields Time_Stamp,
+	/* When the PB_Flag does not equal 0b00 and 0b10, the fields Time_Stamp,
```

Comment:
```suggestion
	/* When the PB_Flag does not equal BT_ISO_START or BT_ISO_SINGLE, the fields Time_Stamp,
```

Reply:
ok, I will make a new PR, tks

Reply:
You dont have to create a new PR, you can just force-push a new version to the same branch and this PR will be updated.

Reply:
> You dont have to create a new PR, you can just force-push a new version to the same branch and this PR will be updated.

got it，tks